.gitignore

```diff
@@ -9,6 +9,7 @@ npm-debug.log
 
 # other
 .next
+/examples/**/yarn.lock
```

Comment:
Done, I though we didn't keep any files there. But apparently we also have a `.babelrc` there 😄